src/odemis/driver/test/orsay_test.py

```diff
@@ -138,26 +141,23 @@ def test_reconnection(self):
         self.oserver._device.MessageConnection.Connection.close()
         self.oserver._device.DataConnection.Connection.close()
         self.oserver._device.MessageConnection.dataConnection.Connection.close()
-        sleep(1)
+        sleep(5)
         while not self.oserver.state.value == model.ST_RUNNING:
             sleep(2)  # wait for the reconnection
 
         # perform some test to check writing and reading still works
-        self.psus._valve.Target = orsay.VALVE_OPEN
-        sleep(1)
-        self.assertTrue(self.psus.power.value)
+        init_state = self.gis_res.temperatureTarget.value
 
-        self.psus._valve.Target = orsay.VALVE_CLOSED
+        test_value = 27
+        self.gis_res.temperatureTarget.value = test_value
         sleep(1)
-        self.assertFalse(self.psus.power.value)
+        self.assertEqual(int(self.gis_res._temperaturePar.Target), test_value)
 
-        self.psus.power.value = True
+        self.gis_res.temperatureTarget.value = 0
```

Comment:
Sorry, I only realise now that this VA is incorrectly named. In the rest of Odemis, we call it `targetTemperature`. Please update the name.